README.md

```diff
@@ -171,6 +171,10 @@ Default: False.
 A Boolean to define whether to configure master.cf to use dovecot as the local delivery agent.  
 Default: False.
 
+#### `use_firma`####
```

Comment:
also it has trailing ####, which is not how the other entries have. I've fixed this and am pushing a new commit here